docs/api/session.md

```diff
@@ -281,7 +281,7 @@ Returns:
 * `event` Event
 * `details` Object
   * `device` [HIDDevice[]](structures/hid-device.md)
-  * `frame` [WebFrameMain](web-frame-main.md)
+  * `origin` string (optional) - The origin that the device has been revoked from.
```

Comment:
this needs a note in breaking-changes.md

Reply:
Ok. I added it to the 20.0 breaking changes because this change is also landing in 20 via #34425.  Also, I didn't say anything about `hid-device-revoked` changing because this is a new feature for 20 which hasn't been released yet.